src/azure-cli/azure/cli/command_modules/databoxedge/generated/_help.py

```diff
@@ -0,0 +1,332 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['databoxedge device'] = """
+    type: group
+    short-summary: Manage device with databoxedge
+"""
+
+helps['databoxedge device list'] = """
+    type: command
+    short-summary: "Gets all the Data Box Edge/Data Box Gateway devices in a resource group. And Gets all the Data Box \
+Edge/Data Box Gateway devices in a subscription."
+    examples:
+      - name: DataBoxEdgeDeviceGetByResourceGroup
```

Comment:
could we refine the example name to be more friendly?

Reply:
same for all

Reply:
all refined under /manual/_help.py